src/gtp_packet.erl

```diff
@@ -767,6 +767,13 @@ encode_imsi(IMSI) ->
     << B:64/bits, _/binary>> = << (encode_tbcd(IMSI))/binary, -1:64 >>,
     B.
 
+decode_imei(Bin) ->
```

Comment:
This is a duplicate functions which are already defined at lines 763 and 766

Reply:
no it's not. Those function are type specific. IMSI and IMEI accidentally use the same encoding, but they are not the same type.

Reply:
Oh... got it. Thanks!